docker/Dockerfile

```diff
@@ -1,12 +1,23 @@
-FROM golang:alpine AS builder
-
-RUN mkdir /src
-ADD . /src
-
-RUN apk add --no-cache make git curl && \
-	cd /src && \
-	mkdir -p output/bin && \
```

Comment:
@dopey this `output/bin` folder never seemed to be used in the rest of the build so I removed this mkdir